### PR TITLE
Front end testing jest

### PR DESCRIPTION
**Pull request by RyanClementsHax**
Will make component tests later, I suspect them to change a lot when we gut out nextjs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 These are set in the your enviromental variables. These must be set before you can run the app
 
 ```
-NODE_ENV = Eviroment name ("development", "testing", "production")
+NODE_ENV = Eviroment name ("development", "test", "production")
```
Comment (RyanClementsHax) [inline README.md]: this was to get jest to work properly

File: src/database/__tests__/firestore.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,189 @@
+import { of } from 'rxjs';
```
Comment (RyanClementsHax) [inline src/database/__tests__/firestore.spec.tsx]: I'm doing a lot of black magic with observables in this file so lmk if you have any questions

File: README.md (hunk 1 above)
Comment (MichaelChan-Pong) [reply README.md]: While you're at it, can we spell `Environment` right? lol
```suggestion
NODE_ENV = Enviroment name ("development", "test", "production")
```

File: src/database/__tests__/firestore.spec.tsx (hunk 2 above)
Comment (MichaelChan-Pong) [reply src/database/__tests__/firestore.spec.tsx]: I thought black magic was prohibited by the church but ok.

**Pull request merged**